docker/docker-compose.yml

```diff
@@ -458,7 +458,6 @@ services:
     environment:
       - BRANDING
       - USE_STRATAGEM
-      - USE_SNAPSHOTS
```

Comment:
AFAIK this is still needed.

Reply:
Only needed on line 355 but not in the setup container.